ops/travis/should-deploy.sh

```diff
@@ -45,6 +45,31 @@ check_overwrite_old() {
     fi
 }
 
+update_endpoints_config_info() {
```

Comment:
This won't work. There's a whitelist of expected keys and you need to pass them all up together. See https://github.com/the-blue-alliance/the-blue-alliance/blob/master/controllers/apiv3/api_admin_controller.py#L28